Apps/Medication/test/unit/Delegates/SalesforceDelegateTests.cs

```diff
@@ -290,16 +283,16 @@ public void ShouldEmptyIfNoContent()
                 .Returns(() => authorizationJwt);
 
             // Setup Http response
-            using HttpResponseMessage httpResponseMessage = new()
-            {
-                StatusCode = HttpStatusCode.NoContent,
-            };
-            Mock<IHttpClientService> mockHttpClient = CreateHttpClient(httpResponseMessage, phn, authorizationJwt.AccessToken);
+            ResponseWrapper? result = null;
+            Mock<ISpecialAuthorityApi> mockSpecialAuthorityApi = new();
+            mockSpecialAuthorityApi
+                .Setup(s => s.GetSpecialAuthorityRequests(It.IsAny<string>(), It.IsAny<string>()))
+                .ReturnsAsync(result);
```

Comment:
This does not test HttpStatusCode.NoContent.  This assumes when there is a NoContent response, null is returned and there is not an exception occurring i.e., 200 and 204 are treated the same when deserializing.

Reply:
I've removed this test as it was obsolete.